### PR TITLE
Auto increase

### PR DESCRIPTION
**Pull request by idanstark42**
Using the tournament status API.
If the API shows no new matches for the team, it will select the first match that wasn't scored.

### CODE REVIEW

File: client/src/services/tournament.js [hunk 1]
```diff
@@ -1,3 +1,5 @@
+const MATCHES_FOR_ALL_TABLES = 2
```
[the remark this answers is not in the document]
Comment (idanstark42) [reply client/src/services/tournament.js]: This would be good

[the remark this answers is not in the document]
Comment (taltaub22) [reply client/src/services/tournament.js]: the usage is as follows: `http://URL:PORT/match/upcoming/table/5/`
Where 5 is the tableId (NOT _id)

[the remark this answers is not in the document]
Comment (idanstark42) [reply client/src/services/tournament.js]: Great. What is the structure of the response?

[the remark this answers is not in the document]
Comment (taltaub22) [reply client/src/services/tournament.js]: A match object

[the remark this answers is not in the document]
Comment (idanstark42) [reply client/src/services/tournament.js]: And if there is none? A `null`?

[the remark this answers is not in the document]
Comment (taltaub22) [reply client/src/services/tournament.js]: I think so........

[the remark this answers is not in the document]
Comment (idanstark42) [reply client/src/services/tournament.js]: Then done

**Pull request merged**